### PR TITLE
Create how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme…

### PR DESCRIPTION
**Pull request by JJM317**
…-Zangpo.md

### CODE REVIEW

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "How to Transform Sickness and Other
+Circumstances"
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:3]: no line break in the middle of a title, please

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - "https://drive.google.com/file/d/1jJKBE2FyvaFtgi82Tj8dBmxtMpFxFdOG/view?usp=sharing"
+formats: [pdf]
+course: thought
+tags:
+  - tibet
+  - mahayana
+  - resilience
+year: 2007
+publisher: "Lotsawa House"
+pages: "2"
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:19]: `pages: 1`

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - "Gyalsé Tokmé Zangpo"
+translator: "Adam Pearcey"
+editor: "Phillippa Sison"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/gyalse-thogme-zangpo/how-transform-sickness"
+drive_links:
+  - "https://drive.google.com/file/d/1jJKBE2FyvaFtgi82Tj8dBmxtMpFxFdOG/view?usp=sharing"
+formats: [pdf]
+course: thought
+tags:
+  - tibet
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:14]: unnecessary

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  - mahayana
+  - resilience
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:16]: `problems`

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+translator: "Adam Pearcey"
+editor: "Phillippa Sison"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/gyalse-thogme-zangpo/how-transform-sickness"
+drive_links:
+  - "https://drive.google.com/file/d/1jJKBE2FyvaFtgi82Tj8dBmxtMpFxFdOG/view?usp=sharing"
+formats: [pdf]
+course: thought
+tags:
+  - tibet
+  - mahayana
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:15]: unnecessary

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+title: "How to Transform Sickness and Other
+Circumstances"
+authors:
+  - "Gyalsé Tokmé Zangpo"
+translator: "Adam Pearcey"
+editor: "Phillippa Sison"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/gyalse-thogme-zangpo/how-transform-sickness"
+drive_links:
+  - "https://drive.google.com/file/d/1jJKBE2FyvaFtgi82Tj8dBmxtMpFxFdOG/view?usp=sharing"
+formats: [pdf]
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:11]: Is this work a poem?  If so it should have a field: `subcat: poetry` 😁

**Review by buddhist-uni — COMMENTED** [review]
all lower case letters in the filename, please 😊

**Comment by JJM317** [comment]
This isn't listed as a poem. Lotsawa House has it listed as 'mind training'
under this author's works, not 'songs and poems' (one of the topics
<https://www.lotsawahouse.org/topics/>in this database. Thoughts?

~Joe


--------------------

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+course: thought
+tags:
+  - problems
+  - resilience
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:14]: Isn't resilience redundant with problems?

File: _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+year: 2007
+publisher: "Lotsawa House"
+pages: "1"
```
Comment (buddhist-uni) [inline _content/essays/how-to-transform-sickness-and-other-circumstances_Gyalse-Tokme-Zangpo.md:17]: I really did mean without the quotes

**Review by buddhist-uni — COMMENTED** [review]
Good enough for me, r.e. not labeling it poetry 😁

But the Capital Letters in the file name still need to be made lower case 😊

**Comment by JJM317** [comment]
Ok, since you really did mean it, I took off the quotes ;)

--------------------

**Review by buddhist-uni — APPROVED** [review]
Looks good! Thanks 😊

**Pull request merged**